src/scripting/game_lua_kernel.cpp

```diff
@@ -3189,8 +3189,8 @@ int game_lua_kernel::intf_get_achievement(lua_State *L)
  * - Arg 2: string - id.
  * - Arg 3: int - the amount to progress the achievement.
  * - Arg 4: int - the limit the achievement can progress by
- * - Ret 1: int - the achievement's current progress after adding amount, -2 if not found, -1 not a progressable achievement (including if it's already achieved)
- * - Ret 2: int - the achievement's max progress, -2 if not found, or -1 not a progressable achievement
+ * - Ret 1: int - the achievement's current progress after adding amount or -1 not a progressable achievement (including if it's already achieved)
+ * - Ret 2: int - the achievement's max progress or -1 not a progressable achievement
```

Comment:
Both lines would be easier to read if the "not" was changed to "if not".

Reply:
Updated.